mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -695,26 +685,18 @@ def get_oois_for_boefje(self, boefje: models.Plugin, organisation: str) -> list[
 
         return oois
 
-    def set_cron(self, item: models.Task) -> str | None:
-        """Override Schedule.set_cron() when a boefje specifies a schedule for
-        execution (cron expression) we schedule for its execution"""
-        # Does a boefje have a schedule defined?
-        plugin = self.ctx.services.katalogus.get_plugin_by_id_and_org_id(
-            utils.deep_get(item.data, ["boefje", "id"]), item.organisation
-        )
-        if plugin is None or plugin.cron is None:
-            return super().set_cron(item)
-
-        return plugin.cron
-
-    def calculate_deadline(self, task: models.Task) -> datetime:
+    def calculate_deadline(self, schedule: models.Schedule) -> models.Schedule:
```

Comment:
Cool! I also noticed this potential win during an analysis of the performance impact of removing the cashes ([comment](https://github.com/minvws/nl-kat-coordination/pull/4108#issuecomment-2681467624)).